WordPress/Tests/NamingConventions/PrefixAllGlobalsUnitTest.1.inc

```diff
@@ -425,3 +425,26 @@ define(
 );
 
 // phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[]
+
+/*
+ * Bad: Issue https://github.com/WordPress/WordPress-Coding-Standards/issues/1733.
+ * 
+ * Short prefixes are not allowed. The errors are triggered
+ * on LINE 1 for the unit-test, because it's the phpcs:set command that is
+ * wrong, not the implementing code.
+ */
+// phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[] a
+function a_do_something(){}
+
+// phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[] aa
+function aa_do_something(){}
+
+// The following line mimicks an empty prefix value.
+// phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[] ,
+function aa_do_something(){}
+
+// phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[] 😊
+function 😊_do_something(){}
+
+// phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[] 😊😊
+function 😊😊_do_something(){}
```

Comment:
Please reset the prefixes to an empty array at the end of the file as otherwise the unit tests in the next test case file may be influenced.

```php
// phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[]
```